pkg/interface/src/views/apps/chat/components/lib/chat-window.tsx

```diff
@@ -267,7 +213,7 @@ export class ChatWindow extends Component<ChatWindowProps, ChatWindowState> {
     const messages: Map<number, Envelope | IMessage> = new Map();
 
     envelopes
-      .sort((a, b) => b.when - a.when)
+      .sort((a, b) => a.when - b.when)
```

Comment:
This is the reason for the DMs being in a weird order. We saw this last time someone tried a flexbox implementation that didn't use column-reverse I think? Or something. We fixed it using the tweak in #2751. Just do that in the root component.